OrbitGl/PickingManager.h

```diff
@@ -7,6 +7,7 @@
 #include <cstring>
 #include <unordered_map>
 #include <vector>
+#include <memory>
```

Comment:
Nit: Since you're already cleaning up the includes :-) `<type_traits>` and `<string>` are also missing. (Please feel free to ignore. Can also be done later since you haven't introduced these issues.)

Reply:
Added, since I'm already cleaning up ;-)